src/components/Contacts/Communication/CommunicationItem/CommunicationItem.tsx

```diff
@@ -0,0 +1,40 @@
+import classNames from "classnames";
+import { ICommunicationItems } from "../items";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+import GroupIcons from "@components/UI/GroupIcons/GroupIcons";
+
+interface CommunicationItemProps extends Omit<ICommunicationItems, "id"> {}
+
+export default function CommunicationItem({
+  title,
+  description,
+  icon,
+}: CommunicationItemProps) {
+  return (
+    <li className={classNames(Styles.item)}>
+      <div className={Styles.groupIcon}>
+        <GroupIcons
+          baseIcon={icon.baseIcon}
+          secondIconShape={icon.secondIconShape}
+          secondIconColor={icon.secondIconColor}
+          gradient={icon.gradient}
+          secondIconSize="sm"
+        />
+      </div>
+      <Link href="">
```

Comment:
Don't leave the empty string in `href` prop value.
Be better if you paste the hash symbol instead
```suggestion
      <Link href="#">
```